jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -629,6 +629,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gce-gci-qa-m60:
```

Comment:
Actually we are deprecating jenkins and moving all ci-jenkins jobs to prow, would you add them to https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml instead?